packages/client/src/components/Conclusion/NewCheckerModal.tsx

```diff
@@ -73,7 +73,7 @@ const NewCheckerModal: React.FC<{
       ]);
 
       // @TODO: We need to find a better solution for restarting the same checker.
-      return (window.location.href = `/${checkerSlug}/vragen-en-conclusie?loadChecker`);
+      window.location.href = `/${checkerSlug}/vragen-en-conclusie?loadChecker`;
```

Comment:
Let's add the typescript signature for all functions. It's now more clear the `return` did nothing.